app.py

```diff
@@ -534,7 +563,11 @@ def paged_results(query, page, per_page, querystring=''):
     '''
     total = query.count()
     last, offset = page_info(query, page, per_page)
-    model_dicts = [o.asdict(True) for o in query.limit(per_page).offset(offset)]
+    if(querystring.find("only_ids") != -1):
+        model_dicts = [o.id for o in query.limit(per_page).offset(offset)]
+    else:
+        model_dicts = [o.asdict(True) for o in query.limit(per_page).offset(offset)]
```

Comment:
Does anyone know how to exclude an attribute from the API output here? I've tried `options(defer('tsv_body'))` already. That doesn't work with `asdict(True)`